src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java

```diff
@@ -108,19 +108,22 @@ public enum ConfigPropertyConstants {
     private final String defaultPropertyValue;
     private final PropertyType propertyType;
     private final String description;
+    private final String
     private final ConfigPropertyAccessMode accessMode;
 
     ConfigPropertyConstants(final String groupName,
                             final String propertyName,
                             final String defaultPropertyValue,
                             final PropertyType propertyType,
                             final String description,
+                            final String teams,
```

Comment:
This will break all enum values in this file, since they don't provide this constructor parameter.

I'd suggest to remove this change, since assigning projects to teams is done in other ways, see comments in the test file.